keylime/cloud_verifier_tornado.py

```diff
@@ -487,10 +487,11 @@ def post(self):
                     # - Name, policy: store policy using name
 
                     ima_policy_name = json_body.get("ima_policy_name")
-                    ima_policy_bundle = json.loads(json_body.get("ima_policy_bundle"))
+                    ima_policy_bundle = json_body.get("ima_policy_bundle")
```

Comment:
A one line fix: `ima_policy_bundle = json.loads(json_body.get("ima_policy_bundle", "{}"))`?

Reply:
True! I'll change it.